lib/go-atscfg/serverunknown.go

```diff
@@ -75,7 +75,7 @@ func MakeServerUnknown(
 		txt += pa.Val + "\n"
 	}
 
-	txt = strings.Replace(txt, `__HOSTNAME__`, *server.HostName+`.`+*server.DomainName, -1)
+	txt = strings.Replace(txt, `__HOSTNAME__`, *server.HostName, -1)
```

Comment:
Nitpick: This could be removed, since t3c-preprocess will do this. But not a huge deal, behaves correctly.